File: Chapter 04/JS/section_4_4_anti_patterns/after_rich_domain_thin_controller/domain/item.js
/**
 * ARCHITECTURE NOTE: A simple data entity. The behavior regarding 
 * how items are priced and discounted is encapsulated inside the 
 * Rich 'Order' model, not here.
 */
class Item {
    constructor() {
        this.price = 0.0;
        this.quantity = 0;
    }
}

module.exports = Item;

File: Chapter 04/Python/section_4_4_anti_patterns/after_rich_domain_thin_controller/domain/item.py
class Item:
    """
    ARCHITECTURE NOTE: A simple data entity. The behavior regarding 
    how items are priced and discounted is encapsulated inside the 
    Rich 'Order' model, not here.
    """
    def __init__(self):
        self.price = 0.0
        self.quantity = 0